### DISCUSSION TITLE
Evennia Engine development discussions

### DISCUSSION THREAD

**Discussion by Griatch** [evennia/evennia#2275] [Evennia Engine Development]
This is a general category for discussing the development of Evennia itself, past and future.
If you have some Evennia-ideas you want to discuss, this could also be the place to hash them out before making them into formal [feature requests](https://github.com/evennia/evennia/issues/new/choose).

### Some useful (online) tools

These can be useful for making diagrams/examples etc to show here: 

- [cloudcraft](https://app.cloudcraft.co) - making server layout diagrams (mainly AWS but works for general use too)
- [AWWboard](https://awwapp.com/#) - online whiteboard with drawing tools for simple diagrams

**Comment by kovitikus** [comment]
Hey, it may be worthwhile to change this category to `Evennia Engine Development`. I really legit confused this category with the General Discussion category, until I read this post and realized that this isn't about developing a MUD with Evennia, but discussing the engine's features and development.

**Reply by Griatch** [reply]
Fair enough, that's a good idea.
